src/connectionTool/utills/LogSaver.java

```diff
@@ -0,0 +1,49 @@
+package connectionTool.utills;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LogSaver {
+
+	private static final String LOG_FILE_DIRECTORY = "log.txt";
+	private static final String DATE_FORMAT = "dd-MM-yyyy HH:mm:ss";
+	public static void appendLog(String text)
+	{
+		File logFile = new File(LOG_FILE_DIRECTORY);
+		if (!logFile.exists())
+		{
+			try
+			{
+				logFile.createNewFile();
```

Comment:
method returns false when file already exists. Is it ok for us to handle this case?